### PR TITLE
Lot 37/verify employer

### PR DESCRIPTION
**Pull request by Webhst**

### CODE REVIEW

File: lib/routes/profile/sub_routes/view_profile/item_tyler_provider.dart [hunk 1]
```diff
@@ -0,0 +1,62 @@
+import 'dart:core';
+import 'package:flutter/material.dart';
+
+class ItemTylerProvider extends StatefulWidget {
+  ItemTylerProvider({
```
Comment (brianschardt) [inline lib/routes/profile/sub_routes/view_profile/item_tyler_provider.dart]: we should change the name of this widget not type and not provider

File: lib/routes/profile/sub_routes/view_profile/verification_screen/verification_screen.dart [hunk 2]
```diff
@@ -0,0 +1,195 @@
...
+                                IconPath.crossBlack,
+                                width: 24,
+                                height: 24,
+                              ),
+                            ),
+                            onTap: () => {
+                              Routes.sailor.navigate(RouteNames.viewprofile)
+                            },
+                          )),
+                      Container(
```
Comment (brianschardt) [inline lib/routes/profile/sub_routes/view_profile/verification_screen/verification_screen.dart]: Lets use the sub_profile_base widget to simplify this.

File: lib/routes/profile/sub_routes/view_profile/verification_screen/verification_screen.dart [hunk 3]
```diff
@@ -0,0 +1,195 @@
...
+                            ),
+                          )),
+                      Column(
+                        children: <Widget>[
+                          TextItemVerification(
+                            right: 220,
+                            left: 0,
+                            top: 20,
+                            title: 'Your Email',
+                            size: 16,
```
Comment (brianschardt) [inline lib/routes/profile/sub_routes/view_profile/verification_screen/verification_screen.dart]: have you tried this on different screen sizes? looks like its not responsive

File: lib/routes/profile/sub_routes/view_profile/item_tyler_provider.dart (hunk 1 above)
Comment (Webhst) [reply lib/routes/profile/sub_routes/view_profile/item_tyler_provider.dart]: done , also that items is used when the user has verified the email , and the other items.

**Pull request closed without merging**